salsah/src/test/scala/org.knora.salsah/browser/SearchAndEditSpec.scala

```diff
@@ -654,11 +654,10 @@ class SearchAndEditSpec extends SalsahSpec {
             // get a list of editing fields
             val editFields = page.getEditingFieldsFromMetadataSection(metadataSection)
 
-            val partOfField = editFields(2)
+            val partOfField = editFields(3)
```

Comment:
incunabula is very prone to errors as:
- [imagebase.js:99](https://github.com/dhlab-basel/Knora/blob/develop/salsah/src/public/js/imagebase.js#L99) sorts properties by gui-order and
- [incunabula-onto.ttl](https://github.com/dhlab-basel/Knora/blob/develop/webapi/_test_data/ontologies/incunabula-onto.ttl) declares `incunabula:page`'s properties `pageOf` and `description` with the same `guiOrder` value.